CMakeLists.txt

```diff
@@ -127,6 +127,47 @@ if(MSVC)
   ADD_DEFINITIONS("-DBOOST_ALL_NO_LIB")
 endif(MSVC)
 
+if(RDK_BUILD_INCHI_SUPPORT)
```

Comment:
Yeah, I agree. Anyway it needs more testing: it is failing on Travis. WIll give it a go as soon as I have a bit of time.